web-performance/README.md

```diff
@@ -0,0 +1,21 @@
+Web Performance
+===============
+
+Web performance refers to the speed in which web pages are downloaded and displayed on the user's web browser. Web performance optimization (WPO), or website optimization is the field of knowledge about increasing web performance.
```

Comment:
Worth splitting lines up at 80 chars limit? Will make it easier in the future to see diff changes to text.

Reply:
This is a great point. I will edit accordingly, thank you!